022-quarkus-properties-config-all/src/main/java/org/acme/GreetingResource.java

```diff
@@ -0,0 +1,78 @@
+package org.acme;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.acme.config.AntagonistConfiguration;
+import org.acme.config.ProtagonistConfiguration;
+import org.acme.converter.KrustyEmail;
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.inject.Inject;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Path("/hello")
+public class GreetingResource{
```

Comment:
```suggestion
public class GreetingResource {
```
the same is on few other places